types/index.d.ts

```diff
@@ -1884,6 +1884,11 @@ export class InlineNotification extends CarbonSvelteComponent {
 
 export class Link extends CarbonSvelteComponent {
   $$prop_def: {
+    /**
```

Comment:
Did you manually add this? It should be generated as part of the build process from running `yarn prepack`.